src/App.tsx

```diff
@@ -1,14 +1,40 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
fix it
Also in this task not asked to add debounce
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/55200530/f8bc9705-daa5-4042-b9fe-498aebe8c07e)